initfiles/componentfiles/configxml/mysqlserver.xsd

```diff
@@ -116,12 +116,19 @@
                     </xs:appinfo>
                 </xs:annotation>
             </xs:attribute>
-            <xs:attribute name="computer" type="computerType" use="required">
+            <xs:attribute name="computer" type="computerType" use="optional">
                 <xs:annotation>
                     <xs:appinfo>
                         <tooltip>The computer on which the MySQL database is deployed.</tooltip>
                         <autogenforwizard>1</autogenforwizard>
                     </xs:appinfo>
+                </xs:annotation> 
+            </xs:attribute>
+            <xs:attribute name="host" type="xs:string" use="optional">
```

Comment:
The idea here is that you can specify a hostname without having to set up a machine in the environment and reference it... but existing components like eclserver referencing a legacy repository will also be affected.

I'll need to see the corresponding LN PR before I can comment on whether this change is safe/correct...